app/backend/express-server.ts

```diff
@@ -0,0 +1,71 @@
+import express from 'express';
+import logger from 'morgan';
+import compress from 'compression';
+import { frameguard, xssFilter, noSniff, ieNoOpen, hsts } from 'helmet';
+import lusca from 'lusca';
+import cookieParser from 'cookie-parser';
+import bodyParser from 'body-parser';
+import formatLogs from '../utils/logging';
+import session from './middleware/session';
+import graphQlMiddleware from './middleware/graphql';
+import ssoMiddleware from './middleware/sso';
+
+const isProd = process.env.NODE_ENV === 'production';
+const port = process.env.PORT || 3000;
+
+const HALF_DAY = 12 * (60 * 60 * 1000);
+const ONE_DAY = 2 * HALF_DAY;
+const TWO_WEEKS = 14 * ONE_DAY;
+const THIRTY_DAYS = 30 * ONE_DAY;
+
+const { json, urlencoded } = bodyParser;
+const { p3p, referrerPolicy } = lusca;
+
+const initExpresss = async () => {
+  const expressServer = express();
+
+  expressServer.use(logger(isProd ? formatLogs : 'dev'));
+  expressServer.use(json());
+  expressServer.use(urlencoded({ extended: false }));
+  expressServer.use(cookieParser());
+
+  // helmet
+  expressServer.use(frameguard());
+  expressServer.use(xssFilter());
+  expressServer.use(noSniff());
+  expressServer.use(ieNoOpen());
+  expressServer.use(
+    hsts({
+      maxAge: TWO_WEEKS,
+      includeSubDomains: true
+    })
+  );
+
+  // lusca
+  expressServer.use(p3p('ABCDEF'));
```

Comment:
This is not a valid P3P policy string. Let's add a card for reviewing the app headers?